src/icalendar/tests/test_with_doctest.py

```diff
@@ -73,6 +73,3 @@ def test_documentation_file(document, zoneinfo_only, env_for_doctest):
 
 def test_can_import_zoneinfo(env_for_doctest):
     """Allow importing zoneinfo for tests."""
-    import pytz
```

Comment:
These three need to stay. The others can go. They are part of the test.

Reply:
Hm. One could probably  use them in there....

Reply:
#650 obsoletes this change, will create a merge conflict. We can thus just merge it and it will not get lost.